examples/views/button/paginator.py

```diff
@@ -16,39 +16,46 @@ def __init__(self, embeds: List[disnake.Embed]):
         # Current embed number.
         self.embed_count = 0
 
+        # Disables previous page button by default.
+        self.last_page.disabled = True
+
```

Comment:
How about we also set the footer in the `__init__` instead of inside the command?